NASA-PDS/harvest
# Use schema from schemaLocation URL

## 🗒️ Summary
Use schema from schemaLocation URL
(See https://github.com/NASA-PDS/pds-registry-mgr-elastic/issues/31)

## ⚙️ Test Instructions
* Delete old registry. Create new registry. (Use the latest version of `registry-manager` on `schema` branch)
* Run harvest with the following data file ([mvn_swe_l2_arc3d_20210102_v04_r03.zip](https://github.com/NASA-PDS/harvest/files/7687711/mvn_swe_l2_arc3d_20210102_v04_r03.zip))
* Output folder should have `missing_fields.txt` and `missing_xsds.txt` files.

## ♻️ Related Issues
https://github.com/NASA-PDS/pds-registry-mgr-elastic/issues/31
https://github.com/NASA-PDS/pds-registry-mgr-elastic/pull/39
